### DISCUSSION TITLE
Drive getting filled with files in Stream Folder

### DISCUSSION THREAD

**Discussion by bolasogbesan** [datalust/seq-tickets#1447] [Help]
Can I delete the stream files in the stream folder as these files are occupying a lot of space on the drive? If yes, is it possible to automate the same?

Thanks

**Comment by liammclennan** [comment]
Hi,

Those files contain the log event data so you can't delete them, but you can control how much space they use by setting [retention policies](https://docs.datalust.co/docs/retention-policies). 

Another good idea is to [setup an API key](https://docs.datalust.co/docs/getting-logs-into-seq#api-keys) per log source and then use the `Minimum level` and `Filter` settings of the API key to filter out events before they are even stored. 

Finally, you can prevent Seq from logging very large events using the 'Raw event body limit' setting, on the `Settings`->`System` page.

Hope that helps.
